scss/6-components/_oisc-warning.scss

```diff
@@ -34,32 +42,36 @@
       background-color: rgba(0, 0, 0, 0.05);
     }
   }
+}
+
+.cads-prose .cads-oisc-warning__title {
```

Comment:
With the change of scope in https://github.com/citizensadvice/public-website/pull/458 we can likely avoid the need for this override?